src/manifests_workflow/input_manifests.py

```diff
@@ -164,7 +169,11 @@ def add_to_cron(self, version: str) -> None:
         with open(jenkinsfile, "r") as f:
             data = f.read()
 
-        cron_entry = f"H 1 * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;TARGET_JOB_NAME=distribution-build-{self.prefix}\n"
+        build_platform = self.BUILD_PLATFORM.get(self.prefix, "linux")
+
+        cron_entry = f"H 1 * * * %INPUT_MANIFEST={version}/{self.prefix}-{version}.yml;" \
```

Comment:
If I were to nitpick, I would make this an array and call `join` with `;\n` to avoid `\`.